apps/admin-gui/src/app/admin/pages/admin-page/admin-consent-hubs/admin-consent-hubs.component.ts

```diff
@@ -31,4 +37,9 @@ export class AdminConsentHubsComponent implements OnInit {
       this.loading = false;
     });
   }
+
+  evaluateConsents(): void {
+    // TODO: Uncomment
+    // this.selection.selected.forEach((hub) => this.consentsManager.evaluateConsentsForConsentHub(hub));
```

Comment:
so uncomment this and I dunno if this way it will work, maybe you will need to rewrite this into recursion
also we need here success notification for sure
Also I spoke with the product owner and he wants to have confirmation window open to confirm that he really wants to evaluate consents for this consents hubs - I would suggest to extend the universal-confirm-dialog